env0/data_configuration_variable.go

```diff
@@ -132,27 +132,14 @@ func dataConfigurationVariableRead(ctx context.Context, d *schema.ResourceData,
 		return err
 	}
 
-	d.SetId(variable.Id)
-	d.Set("name", variable.Name)
-	d.Set("description", variable.Description)
-	d.Set("value", variable.Value)
-	d.Set("is_sensitive", variable.IsSensitive)
-	d.Set("scope", variable.Scope)
-	d.Set("enum", variable.Schema.Enum)
-	d.Set("is_read_only", variable.IsReadonly)
-	d.Set("is_required", variable.IsRequired)
-	d.Set("regex", variable.Regex)
+	if err := writeResourceData(&variable, d); err != nil {
```

Comment:
Now, this is done almost automatically. The fields in variable.Schema are still handled separately (I may try and fix it in a future PR).